package.json

```diff
@@ -4,6 +4,7 @@
   "version": "3.34.1",
   "main": "./website/server/index.js",
   "dependencies": {
+    "@slack/client": "slackhq/node-slack-sdk#2ee794cd31326c54f38c518eef2b9d223327d939",
```

Comment:
I [wrote the Incoming Webhook feature](https://github.com/slackhq/node-slack-sdk/pull/232) for this module, but it hasn't been published to npm yet. When it is, we can switch this out with a version number instead.

We'll need to update shrinkwrap.